### ISSUE TITLE
Add straight drive

### ISSUE THREAD

**Issue by andrewda** [SouthEugeneRoboticsTeam/DeepSpace-2019#51]
Use encoders to drive straight, proportional to driver's input.

**Comment by will-hou** [comment]
Is it okay if I try working on this?

**Comment by andrewda** [comment]
Yes definitely! For now you can bind it to right thumbstick button, might want to change it eventually though. We can talk to Ben today.

**Comment by will-hou** [comment]
Sounds good. Unfortunately I don't think I can make it to robotics today so would you mind letting me know what y'all decide on?

**Comment by andrewda** [comment]
Sure!
